docs/guides/ultimate-guides/torrents/ultimate-intro-to-torrents/index.md

```diff
@@ -0,0 +1,152 @@
+---
+slug: ultimate-intro-to-torrents
+title: The Ultimate Intro to Torrents
+tags: [torrent, tutorial]
+authors: TechSquidTV
+---
+
+# What is a Torrent?
+
+More than likely, you are at least somewhat familiar with torrents. They are a way to download files from the internet.
+But how do they work? What are the benefits of using torrents? How do you use them? Are they safe? Consider the following your guide to _torrents_.
+
+## The Background of BitTorrent
+
+In the late 90s [Bram Cohen](https://en.wikipedia.org/wiki/Bram_Cohen) ([twitter](https://twitter.com/bramcohen)), a software developer, was working for a company called [MojoNation](<https://en.wikipedia.org/wiki/Mnet_(peer-to-peer_network)#MojoNation>) that was trying to create an encrypted decentralized file storage system for commercial-use.
+Unfortunately, the company was faced with severe financial difficulties and would close in 2002.
+Just before then, Cohen would realize that the technology he had been working on, while not a commercial success, could be used to create a new file sharing protocol that could dramatically speed up the process of sharing large files, such as [porn collections](https://www.wired.com/2005/01/bittorrent-2/).
```

Comment:
[Nitpick] Structure-wise, I think it would be helpful to place the second paragraph first. It'll exemplify Cohen's rise and fall a bit clearly.  
```suggestion
Unfortunately, the company was faced with severe financial difficulties and would close in 2002.
Just before then, Cohen would realize that the technology he had been working on, while not a commercial success, could be used to create a new file sharing protocol that could dramatically speed up the process of sharing large files, such as [porn collections](https://www.wired.com/2005/01/bittorrent-2/).
```